it/uk/gov/hmrc/fileupload/UploadFileIntegrationSpec.scala

```diff
@@ -46,11 +41,8 @@ class UploadFileIntegrationSpec extends IntegrationSpec with EnvelopeActions wit
       And("FileScanned")
       sendCommandMarkFileAsClean(MarkFileAsClean(envelopeId, fileId, fileRefId))
 
-      And("I have a valid file attached to the request body")
-      val data = "{}".getBytes
-
-      When(s"I invoke PUT envelope/$envelopeId/files/$fileId/$fileRefId")
-      val response: WSResponse = upload(data, envelopeId, fileId, fileRefId)
+      When(s"StoreFile($envelopeId, $fileId, $fileRefId, 123L) command is sent")
+      val response: WSResponse = sendCommandStoreFile(StoreFile(envelopeId, fileId, fileRefId, 123L))
 
```

Comment:
Upload endpoint has been removed due to upload action now taking place merely in the frontend. Frontend, upon storing file successfully sends StoreFile command to the backend so this command actually tests the new endpoint that was introduced within S3 stuff